### ISSUE TITLE
[FEATURE REQUEST] Debrid-link.fr

### ISSUE THREAD

**Issue by TeaqariaWTF** [LivingWithHippos/unchained-android#253]
I wonder if support can be added for debrid-link.fr

**Comment by LivingWithHippos** [comment]
Right now the app is deeply integrated with real debrid. It would be interesting to make it modular to add support easily for more debird services, but it would probably require quite a big effort, so at the moment it's not doable (time is limited)

**Comment by Invictaz** [comment]
Do all these services have different datacenters? Or is it just a reseller with a different API?

**Comment by TeaqariaWTF** [comment]
> Şu anda uygulama, gerçek debrid ile derinlemesine entegre edilmiştir. Daha fazla debird hizmeti için kolayca destek eklemeyi modüler hale getirmek ilginç olurdu, ancak muhtemelen oldukça büyük bir çaba gerektirecektir, bu nedenle şu anda yapılamaz (zaman sınırlıdır)

So is it possible in the future?

**Comment by LivingWithHippos** [comment]
> So is it possible in the future?

Yeah I'd like to do something like this, but as I said it would be given low priority

**Comment by TeaqariaWTF** [comment]
> > Peki gelecekte mümkün mü?
> 
> Evet böyle bir şey yapmak isterdim ama dediğim gibi düşük öncelik verilecek

It's very gratifying to see that it's not impossible. I hope we get this feature with an update in the near future.
